bundles/org.openhab.persistence.influxdb/src/main/java/org/openhab/persistence/influxdb/internal/FilterCriteriaQueryCreator.java

```diff
@@ -32,21 +32,13 @@ public interface FilterCriteriaQueryCreator {
     String createQuery(FilterCriteria criteria, String retentionPolicy);
 
     default String getOperationSymbol(FilterCriteria.Operator operator, InfluxDBVersion version) {
-        switch (operator) {
-            case EQ:
-                return "=";
-            case LT:
-                return "<";
-            case LTE:
-                return "<=";
-            case GT:
-                return ">";
-            case GTE:
-                return ">=";
-            case NEQ:
-                return version == InfluxDBVersion.V1 ? "<>" : "!=";
-            default:
-                throw new UnnexpectedConditionException("Not expected operator " + operator);
-        }
+        return switch (operator) {
+            case EQ -> "=";
+            case LT -> "<";
+            case LTE -> "<=";
+            case GT -> ">";
+            case GTE -> ">=";
+            case NEQ -> version == InfluxDBVersion.V1 ? "<>" : "!=";
+        };
```

Comment:
Is this skipped on purpose?
```java
default -> throw new UnnexpectedConditionException("Not expected operator " + operator);
```

Reply:
Since `Operator` is an `enum` and all cases are covered, there is no need for `default`.